packages/daemon/src/service/pipeline.ts

```diff
@@ -124,10 +124,9 @@ export class PipelineService {
     ]);
     return results[0];
   }
-
-  async createJob(pipelineId: string): Promise<JobEntity> {
+  async createJob(pipelineId: string, params?: JobParam[]): Promise<JobEntity> {
```

Comment:
```suggestion
  async createJob(pipelineId: string, params: JobParam[]): Promise<JobEntity> {
```

Reply:
We should keep this nullable. The createJob method is also used for pipeline creation which does not involve with job-level param config.